src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -279,6 +280,7 @@ Optional<StudentParticipation> findByExerciseIdAndStudentIdAndTestRunWithEagerSu
                 LEFT JOIN FETCH p.results r
                 LEFT JOIN FETCH r.submission s
                 LEFT JOIN FETCH p.submissions
+                LEFT JOIN FETCH r.assessmentNote
```

Comment:
Do we actually need them every time we call this method?
If yes, please change the name of the method accordingly. If no, please write a new method

Reply:
See [comment above](https://github.com/ls1intum/Artemis/pull/8552/files/c1f600e9649219de794eb43e166da9f4088516ba#r1605801897)